postgres/datadog_checks/postgres/util.py

```diff
@@ -100,6 +100,29 @@ def get_schema_field(descriptors):
     ],
 }
 
+LOCK_METRICS = {
+    'name': 'lock_count',
+    'query': """
+        SELECT mode,
+            locktype,
+            pn.nspname,
+            pd.datname,
+            pc.relname,
+            count(*) AS {metrics_columns}
+        FROM pg_locks l
+        JOIN pg_database pd ON (l.database = pd.oid)
+        JOIN pg_class pc ON (l.relation = pc.oid)
+        LEFT JOIN pg_namespace pn ON (pn.oid = pc.relnamespace)
+        WHERE {relations}
```

Comment:
This line is no longer needed because the relations are not being passed in with the new interface.